Projects/Contracts/Sources/View/CoInsured/CoInsuredProcessingScreen.swift

```diff
@@ -121,6 +129,30 @@ struct CoInsuredProcessingScreen: View {
             Spacer()
         }
     }
+
+    private func missingContractAlert() {
+        vm.store.send(.coInsuredNavigationAction(action: .dismissEditCoInsuredFlow))
+        DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
+            let missingContract = store.state.activeContracts.first { contract in
+                if contract.upcomingChangedAgreement != nil {
+                    return false
+                } else {
+                    return contract.currentAgreement?.coInsured
+                        .first(where: { coInsured in
+                            coInsured.needsMissingInfo
+                        }) != nil
+                }
+            }
+            if missingContract != nil {
+                vm.store.send(
```

Comment:
@sladan-hedvig this doesn't work in the case when you press the info card on home page. Then it's not listening to contract store for some reason and I don't know why. It works if you click info card from the insurance tab.

Reply:
I am not sure what exactly. I think we will need to have huddle after this review.